models/reports/config/generic/registered-patients-by-dob-line-list.json

```
@@ -0,0 +1,14 @@
+{
+    "query": "replace this",
+    "status": "published",
+    "notes": "This report generates a list of all patients that have been registered, including date of registration and which user completed the registration. Patients are listed in chronological order by date of administration. The report also includes key demographic information for each patient. ",
+    "dbSchema": "reporting",
+    "queryOptions": {
+      "defaultDateRange": "24hours",
```

Suggest 7 days as there wouldn't be many births.
```suggestion
      "defaultDateRange": "7days",
```